packages/next/src/client/components/router-reducer/fill-cache-with-data-property.ts

```diff
@@ -11,9 +11,8 @@ export function fillCacheWithDataProperty(
   newCache: CacheNode,
   existingCache: CacheNode,
   flightSegmentPath: FlightSegmentPath,
-  fetchResponse: () => Promise<FetchServerResponseResult>,
-  bailOnParallelRoutes: boolean = false
-): { bailOptimistic: boolean } | undefined {
+  fetchResponse: () => Promise<FetchServerResponseResult>
+) {
```

Comment:
Missing return type, I'm guessing it's now `void`?

```suggestion
): void {
```